src/explore.ts

```diff
@@ -235,3 +233,64 @@ export async function isObjectFile(filePath: string): Promise<boolean> {
     return false;
   }
 }
+
+/**
+ * Gets the codeart symbols for a document.
+ * @param document The document.
+ * @returns The codeart symbols.
+ */
+export function getCodeArtSymbols(
+  document: vscode.TextDocument
+): vscode.DocumentSymbol[] {
+  const items: vscode.DocumentSymbol[] = [];
+  let parent: vscode.DocumentSymbol | undefined;
+
+  for (let line = 0; line < document.lineCount; line++) {
+    const text = document.lineAt(line).text;
+    const match = text.match(/^[0-9a-f]+ <([^>]+)>:/);
```

Comment:
This breaks parsing demangled C++, e.g. this line:
```
000000000000f46e <uchen::Parameters<8000008ul>::~Parameters()>:
```
```suggestion
    const match = text.match(/^[0-9a-f]+ <(.+)>:$/);
```

Reply:
Done

Reply:
Done